kvdb-memorydb/Cargo.toml

```diff
@@ -1,11 +1,12 @@
 [package]
 name = "kvdb-memorydb"
-version = "0.1.0"
+version = "0.1.1"
 authors = ["Parity Technologies <admin@parity.io>"]
 repository = "https://github.com/paritytech/parity-common"
 description = "A key-value in-memory database that implements the  `KeyValueDB` trait"
 license = "GPL-3.0"
+edition = "2018"
 
 [dependencies]
-parking_lot = "0.6"
+parking_lot = "0.9"
```

Comment:
We're on `0.8` in `parity-ethereum` and I have a hunch that this could break things. If you feel like trying this change against the master branch over there, that'd be fantastic.